concent_api/core/message_handlers.py

```diff
@@ -379,49 +388,40 @@ def handle_send_force_get_task_result(client_message: message.concents.ForceGetT
         maximum_download_time
     )
 
-    if Subtask.objects.filter(
-        subtask_id = task_to_compute.compute_task_def['subtask_id'],
-        state      = Subtask.SubtaskState.FORCING_RESULT_TRANSFER.name,  # pylint: disable=no-member
-    ).exists():
-        return message.concents.ServiceRefused(
-            reason = message.concents.ServiceRefused.REASON.DuplicateRequest,
-        )
-
-    elif force_get_task_result_deadline < get_current_utc_timestamp():
+    if not client_message.report_computed_task.timestamp < get_current_utc_timestamp() <= force_get_task_result_deadline:
         logging.log_timeout(
             client_message,
             requestor_public_key,
             force_get_task_result_deadline,
         )
         return message.concents.ForceGetTaskResultRejected(
-            reason    = message.concents.ForceGetTaskResultRejected.REASON.AcceptanceTimeLimitExceeded,
+            reason=message.concents.ForceGetTaskResultRejected.REASON.AcceptanceTimeLimitExceeded,
```

Comment:
if it's too early, should the reson be: `REASON.AcceptanceTimeLimitExceeded`?

Reply:
There is no other `REASON` to choose from.